app/(app)/feedback.tsx

```diff
@@ -1,20 +1,144 @@
+    { icon: '😐', text: 'Okay', value: 1 },
+    { icon: '😊', text: 'Great', value: 2 },
+  ];
 
-const feedback = () => {
   return (
-    <MainContainer preset="scroll">
-      <StyledText>feedback</StyledText>
+    <MainContainer preset="scroll" safeAreaEdges={['top']}>
+      <FeedbackSentModal showModal={showModal} />
+      <View style={styles.FeedBackBannerParent}>
+        <FeedBackBanner />
+        <Pressable style={styles.headerTitle} onPress={() => router.back()}>
+          <AntDesign name="arrowleft" size={24} color={'white'} />
+          <StyledText style={[{ color: colors.whiteConstant }]} size="lg">
+            Feedback
+          </StyledText>
+        </Pressable>
+      </View>
+      <View style={styles.FeedBackFormContainer}>
+        <StyledText size="lg" font="bold" variant="text" style={[styles.FeedBackFormTitle, { color: colors.primary }]}>
+          Your feedback helps us improve
+        </StyledText>
+        <View style={[styles.FeedBackForm, { backgroundColor: colors.background, borderColor: colors.border }]}>
+          <StyledText style={styles.FeedBackFormLabel} size="md">
+            How is/was the event
+          </StyledText>
+          <View style={styles.FeedBackFormRatingContainer}>
+            {ratingStates.map((rating, index) => {
+              return <FeedBackRatingButton rating={rating} key={index} onPress={setSelectedRating} />;
+            })}
+          </View>
+        </View>
+        <TextInput
+          style={[
+            styles.feedbackInput,
+            { backgroundColor: colors.bg, borderColor: colors.borderColor, color: colors.bgInverse },
+          ]}
+          placeholder="Type message here"
+          placeholderTextColor={colors.placeHolder}
+          value={description}
+          onChangeText={setDescription}
+        />
+        <Pressable
+          style={[styles.pressableSubmit, { backgroundColor: colors.assetAccent }]}
+          onPress={() => {
+            setShowModal(true);
+          }}
+        >
+          <StyledText style={[styles.pressableSubmitText]} font="bold">
+            SUBMIT FEEDBACK
+          </StyledText>
+        </Pressable>
+      </View>
     </MainContainer>
   );
 };
 
-export default feedback;
+const styles = StyleSheet.create({
+  headerTitle: {
+    paddingLeft: 23,
+    flexDirection: 'row',
+    columnGap: 22,
+    position: 'absolute',
+    width: '100%',
+    paddingTop: 26,
+    paddingBottom: 20,
+  },
+  FeedBackBannerParent: {
+    maxHeight: 179,
+    overflow: 'hidden',
+    position: 'relative',
+  },
+  FeedBackForm: {
+    flex: 1,
+    marginTop: 29,
+    paddingTop: 17,
+    borderRadius: 10,
+    borderWidth: 1,
+  },
+  FeedBackFormContainer: {
+    paddingTop: 30,
+    width: '90%',
+  },
+  FeedBackFormLabel: {
+    textAlign: 'center',
+    paddingBottom: 30,
+  },
+  FeedBackFormRatingContainer: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'space-evenly',
+    paddingBottom: 57,
```

Comment:
Use paddingBottom 27 as per the design

Reply:
Fixed in commit b48e40c5accb437829e3c40b2f493ab74463d886